### ISSUE TITLE
Update matio

### ISSUE THREAD

**Issue by tbeu** [openmeeg/matio-openmeeg#3]
You might be interested to sync matio with https://github.com/tbeu/matio. :link:

**Comment by papadop** [comment]
Indeed, but what is this repo. Is this a fork of matio, is this the main development repo ? matio-openmeeg tries to track matio master but contains modification to build matio with cmake (so that the build system is portable and there is no need to maintain VC++ build files in addition to autoconf/automake/configure).

What is the intended goal of the tbeu/matio repo ?? Are there any destabilizing changes with respect the matio master ??

**Comment by tbeu** [comment]
This started as my own fork where I collected various fixes and extensions (basically backported from [Modelica/Resources/C-Sources/ModelicaMatIO.c](https://github.com/svn2github/Modelica_StandardLibrary/blob/master/Modelica/Resources/C-Sources/ModelicaMatIO.c)). Since Chris Hulbert recently gave me push/admin rights for the official repo on sf I will push it soon there and prepare to release a v1.6.0. Therefore I also was starting to inspect the list of bugs and patches on the sf site and tried to reproduce and/or fix them. Once I pushed my fork to sf I will keep both repos in sync. My idea is to keep the official repo at sf and use my GitHub fork as my playground for easier collaboration and CI.
